Sdext.tex

```diff
@@ -32,8 +32,9 @@ \section{Debug Mode} \label{debugmode}
 \item No action is taken if a trigger matches.
 \item If \FcsrDcsrStopcount is 0 then counters continue. If it is 1 then
     counters are stopped.
-\item If \FcsrDcsrStoptime is 0 then timers continue. If it is 1 then
-    timers are stopped.
+\item If \FcsrDcsrStoptime is 0 then the time CSR continues to update. If
+    it is 1 then the time CSR will not update. It will resynchronize with
+    mtime after leaving Debug Mode.
```

Comment:
```suggestion
\item If \FcsrDcsrStoptime is 0 then \Rtime continues to update. If
    it is 1 then \Rtime will not update. It will resynchronize with
    \Rmtime after leaving Debug Mode.
```